draft-ietf-jsonpath-base.md

```diff
@@ -1428,7 +1428,8 @@ specification.
 
 A function extension defines a registered name (see {{iana-fnex}}) that
 can be applied to a sequence of zero or more arguments, producing a
-result.
+result. Each registered function name is unique; functions cannot be
```

Comment:
Since we don't define what function overloading might be, this blanket statement could be interpreted to by itself rule out function registrations that allow several types for arguments to a parameter.  I cannot recollect that we explicitly made that decision, although it also seems to be implied from the IANA language ("one for each"...).

Reply:
I've deleted the second part of that sentence rather than continuing to dig a hole.